README.md

```diff
@@ -532,6 +532,66 @@ It returns a Promise that will be fulfilled with `{error?: string; sessionId: st
 **Important! For the transaction to be signed, you will have to use either `SignTransactionsModals` defined above, in the `Prerequisites` section,
 or the `useSignTransactions` hook defined below. If you don't use one of these, the transactions won't be signed**
 
+</details>
+
+<details><summary>
+Sending Batch Transactions
+  </summary>
+
+### Sending Batch Transactions
+
+The API for sending batch transactions is a function called **sendBatchTransactions**:
+
+`import { sendBatchTransactions } from "@multiversx/sdk-dapp/services/transactions/sendBatchTransactions";`
+
+Also, there is a hook that exposes the same functionality: `useSendBatchTransactions`.
+This hook returns a function that can be used to send a batch of transactions and the `batchId` resulted after sending the transactions.
+
+It can be used to send a batch of transactions with minimum information:
+
+  **Important! Each transaction from the batch should be signed and should contain the sender**
+
+```typescript
+const { sessionId, error } = await sendBatchTransactions({
+    transactions: [
+        [{
+          value: '1000000000000000000',
+          data: 'ping',
+          receiver: contractAddress,
+          signature: "asjjsajdjhsajdhjs",
+          sender: "erd1..."
+        }],
+        [{
+          value: '1000000000000000000',
+          data: 'pong',
+          receiver: contractAddress,
+          signature: "asjjsajdjhsajdhjs",
+          sender: "erd1..."
+        }],
+      ],
+      account: 'erd1...',
+      sessionId: "123433533", // unix timestamp
+      apiAddress: "https://api.multiversx.com",
+    });
+```
+
+It returns a Promise that will be fulfilled with
+```
+  {
+    error?: string;
+    batchId: string | null;
+    data?: BatchTransactionsResponseType | null;
+  }
+```
+
+- `batchId` is the transaction's batch id which can be used to track a batch transactions statuses and react to it (is composed of the `sessionId` and the `account addreess`).
```

Comment:
- `batchId` is the transaction's batch id which can be used to track statuses (is composed of the `sessionId` and the `account address`).